vm/src/slots.rs

```diff
@@ -2,8 +2,28 @@ use crate::function::{OptionalArg, PyFuncArgs, PyNativeFunc};
 use crate::pyobject::{IdProtocol, PyObjectRef, PyRef, PyResult, PyValue};
 use crate::VirtualMachine;
 
+bitflags! {
+    pub struct PyTpFlags: u64 {
+        const DEFAULT = 0;
```

Comment:
`bitflags!` generates a "default" in the form of `FlagsType::empty()`, you might want to use that.

Reply:
this is 0 here because i only implemented BAESTYPE. but it can be a non-zero value in CPython. I will add a comment